### ISSUE TITLE
Improve accessibility of contributors selector

### ISSUE THREAD

**Issue by bazzadp** [HTTPArchive/almanac.httparchive.org#1040]
The contributors selector is not accessible to screen readers:

![Contributors team selector screenshot](https://user-images.githubusercontent.com/10931297/87583358-b973e200-c6d3-11ea-9df9-4674ad6c4b67.jpeg)

I see the following problems:
- The filter buttons only used colour to indicate state. They should use `aria-pressed` or similar.
- They do not announce any update to the contributors section using `aria-live` or similar.
- It may be obvious to sighted users as to the state of multiple filters but this is less obvious to non-sighted users. Should we have a summary of all filters available to screen-readers only? Or available to all?

The rest of the site has very good accessibility as far as we are aware and we have a strong [Accessibility Statement](https://almanac.httparchive.org/en/accessibility-statement) that we try our best to honour so we need to fix this.

**Comment by nico3333fr** [comment]
You might use `aria-pressed="true"` on buttons (or `aria-current="true" but `aria-pressed` looks better imho), however, I'd suggest to vocalise only `Filter by team: x contributors` with `aria-live="polite" aria-atomic="true"` and not only the entire content below.

For the filter list, maybe replace it by `ul>li` stuff (so the people will hear "list of X elements"), and a sentence before saying you can use multiple filters. Then a title on each button "filter by _type_", will make it less ambigous (with `aria-pressed`, you know this iw activated or not, with the title, you know this is a filter, and the list will tell you how many there are).
